### ISSUE TITLE
Y-up and G3DB support

### ISSUE THREAD

**Issue by Kerai** [Dancovich/libgdx_blender_g3d_exporter#5]
Without automatic Y-up conversion and possibility to export to G3DB directly, there still needs to be a post-processing step taken before being able to incorporate a model into a game, which makes this even worse solution than using blender->FBX->G3D

I know this ain't easy task, but this exporter isn't very useful without those options. At least the Y-up conversion would make quick prototyping much easier.

**Comment by Dancovich** [comment]
I've been trying to find the time to implement those features for some time now, specially Y-up conversion. For G3DB exporting I created a Java utility that uses libGDX to do that. [Here's the link](https://github.com/Dancovich/libgdx_g3db_converter). I think I could just call it from the Blender exporter directly to avoid the additional post-processing step.

Anyway, the problem right now is finding the time to do all that. This project was just a side project I did for myself and to learn the internals of both libGDX and Blender.

Thanks for the suggestions. I'll leave this issue opened so when I do find the time I'll post here so you'll get reported back.
